src/settings/Settings.js

```diff
@@ -160,6 +160,12 @@ export const SETTINGS = {
         displayName: _td("Show info about bridges in room settings"),
         default: false,
     },
+    "feature_invite_only_padlocks": {
+        isFeature: true,
+        supportedLevels: LEVELS_FEATURE,
+        displayName: _td("Show padlocks on invite only rooms"),
+        default: true,
```

Comment:
oh, I think we've been bitted by this before: does this actually work?

Reply:
It seemed to work in local testing from a config value and via the UI... Hopefully that's the main cases...